wsl-setup

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -eu
+
+# get_first_interactive_uid returns first interactive non system user uid with uid >=1000.
+function get_first_interactive_uid() {
+  getent passwd | egrep -v '/nologin|/false|/sync' | sort -t: -k3,3n | awk -F: '$3 >= 1000 { print $3; exit }'
+}
+
+# create_regular_user prompts user for a username and assign default WSL permissions.
+# First argument is the prefilled username.
+function create_regular_user() {
+  local default_username="${1}"
+
+  DEFAULT_GROUPS='adm,cdrom,sudo,dip,plugdev'
+
+  # Prompt from the username prefilled with the Windows one.
+  read -e -p 'Create a default Unix user account: ' -i "${win_username}" username
+
+  # Create the user and change its default groups.
+  if ! /usr/sbin/adduser --quiet --gecos '' "${username}"; then
+    return
+  fi
+  if ! /usr/sbin/usermod "${username}" -aG "${DEFAULT_GROUPS}"; then
+    # Failure: try to delete the user.
+    /usr/sbin/deluser "${username}"
+  fi
+}
+
+
+echo "Provisioning the new WSL instance $(wslpath -am / | cut -d '/' -f 4)"
+echo "This might take a while"
+
+# Read the Windows user name.
+win_username=$(powershell.exe -NoProfile -Command '$Env:UserName') 2>/dev/null || true
+win_username="${win_username%%[[:cntrl:]]}"
+
+# Copy the Ubuntu font into Windows filesystem, assuming the fonts-ubuntu package is seeded.
+local_app_data=$(powershell.exe -NoProfile -Command '$Env:local_app_data') 2>/dev/null || true
+local_app_data=$(wslpath -au "${local_app_data%%[[:cntrl:]]}") 2>/dev/null || true
+
+if [ -n "${local_app_data}" ]; then
```

Comment:
The output of `wslpath -au ` will never be empty. If the argument is empty, it prints `/`.